.github/workflows/pip.yml

```diff
@@ -81,6 +81,6 @@ jobs:
       with:
         name: artifact
         path: dist
-    - uses: pypa/gh-action-pypi-publish@master
-      with:
-        password: ${{ secrets.PYPI_TOKEN }}
+    # - uses: pypa/gh-action-pypi-publish@master
+    #   with:
+    #     password: ${{ secrets.PYPI_TOKEN }}
```

Comment:
The whole `upload_all` job is gated by
```yaml
if: github.event_name == 'release' && github.event.action == 'published'
```
So, unless you're making a release, it's not going to be triggered. No need to comment it.